command.go

```diff
@@ -685,11 +685,22 @@ func (cmd *ZSliceCmd) readReply(cn *pool.Conn) error {
 
 //------------------------------------------------------------------------------
 
+// Scanner is a ScanCmd with Iterator capabilities
+type Scanner struct {
+	*ScanCmd
+	c *commandable
+}
```

Comment:
We can do it in v4, but I don't see why you need it. Just add new private field to existing struct...